src/panels/lovelace/common/directives/long-press-directive.ts

```diff
@@ -120,6 +124,20 @@ class LongPress extends HTMLElement implements LongPress {
       this.timer = undefined;
       if (this.held) {
         element.dispatchEvent(new Event("ha-hold"));
+      } else if (element.hasDblClick) {
+        if (this.nbClicks === 0) {
+          this.nbClicks += 1;
+          this.dblClickTimeout = window.setTimeout(() => {
```

Comment:
What about https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event

Reply:
cool. I'll check it out

Reply:
So this is nice, but I would still need a timeout to not fire a regular click event is my understanding. I did find a way while researching this using the MouseEvent.detail though that makes the logic a bit simpler.